src/test_projects/example/data_models/model_output.py

```diff
@@ -1,28 +1,13 @@
+from typing import List
 from hypergol import BaseData
 
 
 class ModelOutput(BaseData):
 
-    def __init__(self, articleId: int, sentenceId: int, inputs: object, outputs: object, targets: object):
+    def __init__(self, articleId: int, sentenceId: int, posTags: List[str]):
```

Comment:
positionTags? not sure this needs to be shortened

Reply:
no, pos=Part-Of-Speech, standard linguistic abbreviation